NuRadioReco/modules/LOFAR/planeWaveDirectionFitter_LOFAR.py

```diff
@@ -16,6 +16,55 @@
 from NuRadioReco.modules.base.module import register_run
 from NuRadioReco.modules.LOFAR.beamforming_utilities import geometric_delay_far_field, lightspeed
 
+from NuRadioReco.modules.LOFAR.pipelineVisualizer_LOFAR import check_for_good_ant
```

Comment:
This looks good @PhilippLaub , however I am a bit confused at the check_for_good_ant being a function of the pipelineVisualizer? Would it not be cleaner to make this function part of an "earlier" module within the pipeline? I think it is confusing to have something as important be part of a plotting module. @MijnheerD what do you think?

Reply:
We can either put it in the plane wave fitter or we could think about introducing a e.g. utilities_LOFAR module where we store things like this.

Reply:
if the good antennas are a product of the planewave fitter, the function should be in the planewavefitter (imo)